src/plugins/intel_cpu/src/nodes/memory.cpp

```diff
@@ -861,7 +861,7 @@ void MemoryInput::runStatic(dnnl::stream strm) {
 
 void MemoryInput::resolveInPlaceEdges(Edge::LOOK look) {
     if (!(look & Edge::LOOK_UP)) {
-        Node::resolveInPlaceEdges(look);
+        ov::intel_cpu::node::Input::resolveInPlaceEdges(look);
```

Comment:
What check is it?

Reply:
Discussed offline, it is [bugprone-parent-virtual-call](https://clang.llvm.org/extra/clang-tidy/checks/bugprone/parent-virtual-call.html)